pkg/cloudprovider/provider/azure/types/cloudconfig.go

```diff
@@ -35,7 +35,7 @@ type CloudConfig struct {
 	RouteTableName             string  `json:"routeTableName"`
 	SecurityGroupName          string  `json:"securityGroupName" yaml:"securityGroupName"`
 	PrimaryAvailabilitySetName string  `json:"primaryAvailabilitySetName"`
-	VnetResourceGroup          *string `json:"vnetResourceGroup,omitempty"`
+	VnetResourceGroup          string  `json:"vnetResourceGroup"`
```

Comment:
```
INFO [runner] linters took 16.147422301s with stages: goanalysis_metalinter: 14.441121508s, unused: 1.692719018s
pkg/cloudprovider/provider/azure/types/cloudconfig.go:31: File is not `gofmt`-ed with `-s` (gofmt)
        ResourceGroup              string  `json:"resourceGroup"`
        Location                   string  `json:"location"`
        VNetName                   string  `json:"vnetName"`
        SubnetName                 string  `json:"subnetName"`
        RouteTableName             string  `json:"routeTableName"`
        SecurityGroupName          string  `json:"securityGroupName" yaml:"securityGroupName"`
        PrimaryAvailabilitySetName string  `json:"primaryAvailabilitySetName"`
        VnetResourceGroup          string  `json:"vnetResourceGroup"`
        UseInstanceMetadata        bool    `json:"useInstanceMetadata"`
```

Reply:
sorry

Reply:
No worries, that's why we have CI and lint automated :D